core/src/main/java/greencity/controller/UserController.java

```diff
@@ -420,6 +420,7 @@ public ResponseEntity<UserProfileStatisticsDto> getUserProfileStatistics(
         @ApiResponse(responseCode = "200", description = HttpStatuses.OK),
         @ApiResponse(responseCode = "400", description = HttpStatuses.BAD_REQUEST),
         @ApiResponse(responseCode = "401", description = HttpStatuses.UNAUTHORIZED),
+            @ApiResponse(responseCode = "404", description = HttpStatuses.NOT_FOUND),
```

Comment:
Delete unnecessary free-space before @ApiResponse(...) and coma "," in the end.

Reply:
Roman, thanks for review. I have accepted the remarks and corrected them.